src/context/Store.js

```diff
@@ -43,7 +44,8 @@ const Store = ({children}) => {
     users: React.useReducer(usersReducer, usersState),
     domains: React.useReducer(domainReducer, domainState),
     initialStartup: React.useReducer(initialStartupReducer, initialStartupState),
-    comments: React.useReducer(commentReducer, commentState)
+    comments: React.useReducer(commentReducer, commentState),
+    chat: React.useReducer(chatReducer, chatState)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `chat` reducer to the root reducer is a significant change. Ensure that all components using this context are updated accordingly to handle the new state and dispatch functions provided by this reducer.
```diff
+    chat: React.useReducer(chatReducer, chatState)
```
This change should be thoroughly tested to ensure it doesn't introduce any unexpected behavior in the application.


<!-- This is an auto-generated comment by OSS CodeRabbit -->